src/components/interface/PendingTransaction/PendingTransaction.tsx

```diff
@@ -48,8 +46,6 @@ const PendingTransaction: React.FunctionComponent<PendingTransactionProps> = ({
   liquidityAction,
   isEditingMargin,
   showNegativeNotional,
-  isFCMSwap,
```

Comment:
noticed few more fcm stuff 
<img width="1173" alt="image" src="https://user-images.githubusercontent.com/11079632/202381595-5b9876fa-fe08-46d0-958f-9e797b86361b.png">
shall we remove this as well?

Reply:
That's mandatory for the object constructors.